texthero/preprocessing.py

```diff
@@ -1,15 +1,22 @@
 """
 Preprocess text-based Pandas DataFrame.
 """
-
 import re
 import string
+from typing import Set, Optional
+
+import numpy as np
+import pandas as pd
 import unidecode
 from nltk.corpus import stopwords
-from nltk.stem import PorterStemmer
-from nltk.stem import SnowballStemmer
-import pandas as pd
-import numpy as np
+from nltk.stem import PorterStemmer, SnowballStemmer
+from spacy.lang.en import stop_words as spacy_en_stop_words
+
+
+class StopWordLists:
```

Comment:
is this preferable to a separate `stopwords.py` file with top-level constants?
also might want to lazy load these vars if possible

Reply:
Thank you for the request! Great.

Maybe is not necessary as there are just two stopwords constant, right? (spacy + nltk).

I will integrate the pull request shortly, first I need to merge the `test_preprocessing` as now it uses `unittest`.

Reply:
in which case i think its preferable to have them as top-level constants instead of class vars. Lemme know if you disagree, otherwise will make the change.

Have you considered `pytest` instead of `unittest`? Also what was the reason to choose `yapf` over [black](https://github.com/psf/black)?

Reply:
Agree on top-level constants.

`pytest` vs `unittest`: I just picked `unittest` without spending too much time deciding. Do you think we should switch to pytest? I'm open to other proposals as long there is a concrete difference.

`black` vs `yapf`. I picked `yapf` as the programmer does not have to make many decisions regarding code formatting. Looks like black is about the same. Do you think we should switch to black? I'm fine with that as well in case there is any valuable reason to chose it.